src/main/java/ru/yandex/qatools/allure/jenkins/AllureReportPublisher.java

```diff
@@ -78,9 +78,12 @@
 
     private Boolean includeProperties;
 
+    private boolean allureReportDisabled;
+
     @DataBoundConstructor
-    public AllureReportPublisher(@Nonnull List<ResultsConfig> results) {
+    public AllureReportPublisher(@Nonnull List<ResultsConfig> results, boolean allureReportDisabled) {
```

Comment:
Use getter and setter instead of constructor. 
In this case you will brake pipeline integration.